### PR TITLE
feat(eth): add wrapped token (WDOGE) ABI files and instance address with some examples

### PR DESCRIPTION
**Pull request by DinshawKothari**
Ticket: BG-59996

[BG-59996]: https://bitgoinc.atlassian.net/browse/BG-59996?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: eth/examples/WrappedToken/transfer.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { getContractsFactory } from '../../../src/index';
+/** This example should provide just a method */
```
Comment (sachushaji) [inline eth/examples/WrappedToken/transfer.ts:2]: What does this comment mean in the examples?

File: src/eth/methods/methods.ts [hunk 2]
```diff
@@ -53,7 +53,7 @@ export class EthMethod implements Method {
         types.push(tuple);
         const abiElement = 'function ' + this.name + '(' + tuple + ')';
         const iface = new ethers.utils.Interface([abiElement]);
-        const methodId = iface.functions[this.name].sighash;
+        const methodId = iface.getSighash(iface.functions[this.name])
```
Comment (sachushaji) [inline src/eth/methods/methods.ts:56]: Seems unit tests are failing. Some of them are already failing due to #40. the pipeline may not be configured correctly unfortunately.

**Review by sachushaji — CHANGES_REQUESTED** [review]

File: package-lock.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bitgo/smart-contracts",
-  "version": "2.3.0",
+  "version": "2.4.0",
```
[the remark this answers is not in the document]
Comment (DinshawKothari) [reply package-lock.json:3]: It is already 2.4.0 in package.json

File: eth/examples/WrappedToken/transfer.ts (hunk 1 above)
Comment (DinshawKothari) [reply eth/examples/WrappedToken/transfer.ts:2]: Will remove it.

File: src/eth/methods/methods.ts (hunk 2 above)
Comment (mullapudipruthvik) [reply src/eth/methods/methods.ts:56]: hey just checkout to this commit https://github.com/BitGo/smart-contracts/pull/40/commits/6dae62b075548d97c706161e86ce0bd00e69a0a0 and testcases are passing until this point
I believe this pr https://github.com/BitGo/smart-contracts/pull/44/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519L4133 introduced major ether library upgrade hence tests are failing now

Comment (mullapudipruthvik) [reply src/eth/methods/methods.ts:56]: adding `"ethers": "4.0.48"` in packages.json and using `iface.functions[this.name].sighash;` worked for me

Comment (DinshawKothari) [reply src/eth/methods/methods.ts]: Done

**Review by sachushaji — APPROVED** [review]
LGTM, Nice work @DinshawKothari

**Pull request merged**